decidim-core/spec/system/editor_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Editor", type: :system do
+  let!(:organization) { create(:organization) }
+  let(:content) { "" }
+  let(:toolbar) { "full" }
+
+  before do
+    spec = self
+    request_context = {
+      "decidim.current_organization" => organization
+    }
+    editor_config = {
+      toolbar: toolbar,
+      content: content
+    }
+
+    # Map a special route for rendering the editor
```

Comment:
why not create a system example for existing views with the editor?

Reply:
I was thinking about it but it would've required changes to so many specs, so I thought this is easier.

I also noticed there is no test currently which tests the rich text editors when the records are being edited. The rich text editors are only tested for the "new record" views (if you do a global search with `having a rich text editor`). And that test does not even do anything particularly useful in the context of this bug, as it only checks that the editor container is created with the correct data attribute.

This particular case would have to be tested when an existing record is edited because it requires the content to be already inserted to the view when the editor is created. If you write a single shared example, you will still need to setup the records + organization settings for each test case separately as there is no current tests for rich text editors when the records are being edited.

So, writing rich text editing test cases for 11 different record views vs writing 1 test which tests this particular bug I am fixing. I chose the latter.

Reply:
Ok, let's have this fixed but can you apply the test to "Settings/Configuration/Admin terms of use body" for example, instead of creating a dummy controller?

This way it will be already tested in a more "realistic" case.

Reply:
Moved there by b4b2d32d87a1cbbf06706409acd254ed467c409d.